deploy/helm/aiobs-stack/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
```

Comment:
What about moving aiobas-app and aiobs-infra to the folder under aiobs-stack? This would reflect the relationship logically if this is feasible.

Reply:
@jianrongzhang89 thanks rebases my PR please review again.